internal/sdkv2provider/data_source_jwks_from_certificate.go

```diff
@@ -0,0 +1,164 @@
+package sdkv2provider
+
+import (
+	"context"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"encoding/pem"
+	"math/big"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type Keys struct {
+	Keys []Key `json:"keys"`
+}
+
+type Key struct {
+	Kty    *string  `json:"kty"`
+	X5C    []string `json:"x5c"`
+	N      *string  `json:"n"`
+	E      *string  `json:"e"`
+	Kid    *string  `json:"kid"`
+	X5T256 *string  `json:"x5t#256"`
+	X5Dn   *string  `json:"x5dn"`
```

Comment:
What is this field? https://www.rfc-editor.org/rfc/rfc7517#section-4

Reply:
Removed